image/templates/helm/stackrox-central/templates/_init.tpl.htpl

```diff
@@ -139,6 +139,14 @@
 {{ include "srox.autoSensePodSecurityPolicies" (list $) }}
 [<- end >]
 
+[< if .Operator >]
+{{ $_ := set $env "installMethod" "rhacs-operator" }}
+[< else if not .KubectlOutput >]
+{{ $_ := set $env "installMethod" "helm" }}
+[< else >]
+{{ $_ := set $env "installMethod" "manifest" }}
```

Comment:
nit: logically, this made more sense to me reading through:
```suggestion
[< else if .KubectlOutput >]
{{ $_ := set $env "installMethod" "manifest" }}
[< else >]
{{ $_ := set $env "installMethod" "helm" }}
```

Reply:
Agreed, the reason put it as is was to mirror https://github.com/stackrox/stackrox/blob/master/image/templates/helm/stackrox-secured-cluster/internal/defaults/30-base-config.yaml.htpl#L14-L20. Can change it tho.